rust/noosphere-ns/src/bin/bootstrap_nns/cli.rs

```diff
@@ -0,0 +1,50 @@
+#![cfg(not(target_arch = "wasm32"))]
+
+use clap::{Parser, Subcommand};
+use serde::Deserialize;
+use std::path::PathBuf;
+
+#[derive(Parser)]
+#[clap(name = "bootstrap_nns")]
+pub struct CLI {
+    #[clap(subcommand)]
+    pub command: CLICommand,
+}
+
+#[derive(Subcommand)]
+pub enum CLICommand {
+    Install,
+    Uninstall,
```

Comment:
Looking into systemd daemons that we can un/register via this CLI for our VPS.

Reply:
In the interest of separating concerns, is it possible to set up a daemon separately from the CLI implementation? Or does there have to be a special code path when running as a daemon?

Reply:
Could move it to our "devops scripts", as we mostly just need to drop a text config file pointing at this executable and run a systemd command, so ya

Reply:
Will remove these stubs -- I think I'll leave `run` as a subcommand, as there a handful of other things we can do here that'd be valuable (querying, maybe keygen stuff, if kicking daemon-ish stuff to systemd then maybe there aren't many more commands here)